src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -68,19 +81,20 @@ public PersonCard(ReadOnlyPerson person, int displayedIndex) {
      */
     private static String obtainTagColors(String tagValue) {
 
+        int randomisedIndex;
```

Comment:
I noticed that randomisedIndex is only used in the loop. It's good practice to narrow the scope and clean up the namespace by declaring and initialise randomisedIndex inside the loop.